### PR TITLE
Refactored - codeblocks now make use of the bot.utils.messages util.

### LINKED ISSUE #105: Use wait_for_deletion for codeblocks
**Originally posted by Leon Sandøy:**

In #73, we created a util to handle waiting for reaction and then deleting a message in `bot.utils.messages`. This should be used for codeblock messages as well.

### PR DESCRIPTION
**Pull request by heavysaturn**
Closes #105

### CODE REVIEW

File: bot/cogs/bot.py [hunk 1]
```diff
@@ -342,7 +343,11 @@ def has_bad_ticks(self, msg: Message):
                         howto_embed = Embed(description=howto)
                         bot_message = await msg.channel.send(f"Hey {msg.author.mention}!", embed=howto_embed)
                         self.codeblock_message_ids[msg.id] = bot_message.id
-                        await bot_message.add_reaction(Emojis.cross_mark)
+
+                        # Create
```
Comment (jchristgit) [inline bot/cogs/bot.py]: Did you forget to type this out?

Comment (MarkKoz) [inline bot/cogs/bot.py]: I think the code is adequately self-descriptive anyway.

```suggestion
```

Comment (MarkKoz) [reply bot/cogs/bot.py]: I think the code is adequately self-descriptive anyway.

```suggestion
```

Comment (heavysaturn) [reply bot/cogs/bot.py]: agreed

Comment (heavysaturn) [reply bot/cogs/bot.py]: and yeah oops I guess I forgot to write it

File: bot/utils/messages.py [hunk 2]
```diff
@@ -7,13 +7,15 @@
...
+from bot.constants import Emojis
+
 MAX_SIZE = 1024 * 1024 * 8  # 8 Mebibytes
 
 
 async def wait_for_deletion(
     message: Message,
     user_ids: Sequence[Snowflake],
-    deletion_emojis: Sequence[str] = ("❌",),
+    deletion_emojis: Sequence[str]=(Emojis.cross_mark,),
```
Comment (MarkKoz) [inline bot/utils/messages.py]: `E252 missing whitespace around parameter equals`

```suggestion
    deletion_emojis: Sequence[str] = (Emojis.cross_mark,),
```

**Pull request merged**